app/src/main/java/com/univoice/data/datasource/SignUpDataSource.kt

```diff
@@ -1,9 +1,12 @@
 package com.univoice.data.datasource
 
 import com.univoice.data.dto.BaseResponse
+import com.univoice.data.dto.request.RequestCheckEmailDto
 import com.univoice.data.dto.request.RequestDepartmentDto
 
 interface SignUpDataSource {
     suspend fun postUniversityNames(): BaseResponse<List<String>>
     suspend fun postDepartments(requestDepartmentDto: RequestDepartmentDto): BaseResponse<List<String>>
+    suspend fun postEmail(requestCheckEmailDto: RequestCheckEmailDto): BaseResponse<List<String>>
```

Comment:
P1 : 아무것도 반환하지 않아야하니까..!
```suggestion
    suspend fun postEmail(requestCheckEmailDto: RequestCheckEmailDto): BaseResponse<Unit>
```

Reply:
반영완!